pkg/rpcclient/wsclient.go

```diff
@@ -399,19 +399,15 @@ func (c *WSClient) performUnsubscription(id string) error {
 	return nil
 }
 
-// SubscribeForNewBlocks adds subscription for new block events to this instance
-// of the client. It can be filtered by primary consensus node index and/or block
-// index allowing to receive blocks since the specified index only, nil value is
-// treated as missing filter.
-func (c *WSClient) SubscribeForNewBlocks(primary *int, sinceIndex, tillIndex *uint32) (string, error) {
```

Comment:
We should keep it for a while (in deprecated manner), there is already existing code that uses it.